i18n/de/docusaurus-plugin-content-docs/current/toolbox/accessibility_indicators/closest_average.md

```diff
@@ -6,24 +6,28 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import thematicIcon from "/img/toolbox/data_management/join/toolbox.webp";
 
-# Heatmap - Closest Average
+# Heatmap - Durchschnitt Reisezeit
 A color-coded map to visualize the average travel time to points (such as [POI](../../../further_reading/glossary#point-of-interest-poi "What is a POI?")) from surrounding areas.
 
 ## 1. Explanation
 
-Visualized as a color-coded hexagonal grid, the heatmap takes into account real-world transport and street networks to compute travel times. After specifying a *routing type* (Walk, Bicycle, etc.), *opportunity layer* and *travel time limit*, the result will display a color-coded hexagonal grid for all areas accessible under these conditions. The color scale refers to average travel time.
+Eine farbkodierte Karte zur Visualisierung der durchschnittlichen Reisezeit zu Punkten (wie [POI](../../../further_reading/glossary#point-of-interest-poi "Was ist ein POI?")) aus umliegenden Gebieten.
+
+## 1. Erklärung
```

Comment:
Hey @LucasLomnicki, thanks for working on this!

I think there's a small issue with the formatting here (the old English text seems to still exist above and the heading ## 1. is there twice). Would you be able to fix this please? Thanks!